packages/auth/src/app.ts

```diff
@@ -137,10 +137,7 @@ export class App {
     // Token management
     this.publicRouter.post('/auth/introspect', accessTokenRoutes.introspect)
 
-    this.publicRouter.post('/auth/token/:id', (ctx: AppContext): void => {
-      // TODO: tokenService.rotate
-      ctx.status = 200
-    })
+    this.publicRouter.post('/auth/token/:id', accessTokenRoutes.rotate)
```

Comment:
To make the usage of this endpoint described in https://github.com/interledger/rafiki/pull/375/files#r911538130 not confusing (as I regretfully have written it), maybe we can change the parameter from `id` to `managementId`.